### PR TITLE
feat: Wearable Sets Page with Total cost calculated with Baazaar Floor Price

### PR DESCRIPTION
**Pull request by knyumatt**
# Adding feature: Wearable Sets Page with Total cost calculated with Baazaar Floor Price

The location of the feature: `http://localhost:3005/market/set`

Screenshot 2022-02-14 at 9 02 47 PM

### Notes

1. Feel free to suggest UI changes as I know the UI design may not be fancy enough
2. `src/components/Items/ERC1155/ERC1155.js` and `src/components/Items/Wearable/Wearable.js` are refactored which may affect other pages
3. Only sorting on Total GHST cost is implemented. Feel free to suggest any other sorting
4. There are no way to navigate to the page from the UI at the moment, please suggest where to put the button to navigate to the Wearable Sets page

### CODE REVIEW

File: src/components/Items/ERC1155/ERC1155.js [hunk 1]
```diff
@@ -74,8 +75,12 @@ export default function ERC1155({children, item}) {
...
                                 <Typography variant='subtitle2'>
                                     {
-                                        last.price === 0 && !item.priceInWei ? '???' :
-                                        commonUtils.formatPrice((last.price && item.balance) ? (last.price * item.balance) : parseFloat(web3.utils.fromWei(item.priceInWei)))
+                                        (item.priceInWei && (item.priceInWei === Infinity ? '???' : item.priceInWei)) || 
+                                        (last?.price === 0 && !item.priceInWei 
+                                            ? '???'
+                                            : commonUtils.formatPrice((last?.price && item.balance)
+                                                ? (last?.price * item.balance)
+                                                : parseFloat(web3.utils.fromWei(item.priceInWei))))
```
Comment (andkoval) [inline src/components/Items/ERC1155/ERC1155.js:83]: need to review this price formatting horror :)
it breaks in some other places.
Screenshot 2022-02-15 at 12 23 39

**Review by andkoval — CHANGES_REQUESTED** [review]
@knyumatt first of all awesome work, thanks for your contribution.

1. Looks simple and good for me
2. Some issues with price formatting occurring in wearables cards
3. Sets BRS, Traits
4. I would add it in `/market` sidebar for now, there are some plans to improve market routing since rn it's only 1 page with all data coming at once.

I think we can publish this page after this checklist:

- [ ] add link to `/market` sidebar
- [ ] fix wearable cards price formatting issue
- [ ] add ability to navigate to available baazaar listing for each item

**Do you have the desire to do these adjustments by yourself? Or should we take care of it?**

File: src/components/Items/ERC1155/ERC1155.js (hunk 1 above)
Comment (knyumatt) [reply src/components/Items/ERC1155/ERC1155.js:83]: I will address this issue first. Sorry that I am not familiar with the code base

**Comment by knyumatt** [comment]
@andkoval Thank you for reviewing. 
The last commit has addressed the formatting and the route.

For 
- [ ] add link to `/market` sidebar
- [ ] add ability to navigate to available baazaar listing for each item
, I am not sure about the expectation. Perhaps you guys can handle it better than me.

**Comment by dudendy** [comment]
@knyumatt sorry I missed your answer somehow. Thanks for your work, we will finish this page.

**Pull request closed without merging**